R/label.R

```diff
@@ -1,18 +1,40 @@
 #' Assign Variable Label
 #'
 #' Assigns variable label from a variable level metadata to a given data frame.
+#' This function will give an error if a label is presented that is greater than
+#' 40 characters which isn't allowed in XPT v5. If labels aren't present for the
```

Comment:
```suggestion
#' This function will give detect if a label is greater than
#' 40 characters which isn't allowed in XPT v5. If labels aren't present for the
```
The function only errors when we tell it to.  If we set as warn or message then it just prints to the console the warning or the message